README.md

```diff
@@ -197,7 +197,73 @@ data-validation (--verbose or -v) (--log-level or -ll) validate row
   [--filter-status or -fs STATUSES_LIST]
                         Comma separated list of statuses to filter the validation results. Supported statuses are (success, fail). If no list is provided, all statuses are returned.
 ```
+#### Generate Table Partitions for Large Table Row Validations
 
+Below is the command syntax for Generating Tabe Partitions for performing Row Validation on large Tables.
+The following command splits Row Validation into multiple steps by generating multiple Config files which can then be run in Parallel or independently.
+
+The command takes the same parameters as required for `Row Validation` *plus* few commands to implement the partitioning logic.
```

Comment:
Since we already describe the flags needed below, I don't think we need lines 207-213. We should keep the note that only monotonically increasing keys are supported though.

Reply:
Done